packages/public/server/src/module/CacheInvalidator/config/module.config.php

```diff
@@ -42,49 +42,4 @@
                 __NAMESPACE__ . '\Factory\CacheOptionsFactory',
         ],
     ],
-    //    'cache_invalidator' => [
-    //        'invalidators' => [
-    //            'factories' => [
-    //                __NAMESPACE__ . '\Invalidator\NavigationStorageInvalidator' => __NAMESPACE__ . '\Factory\NavigationStorageInvalidatorFactory',
-    //                __NAMESPACE__ . '\Invalidator\StrokerStorageInvalidator'    => __NAMESPACE__ . '\Factory\StrokerStorageInvalidatorFactory',
-    //                __NAMESPACE__ . '\Invalidator\RepositoryStorageInvalidator' => __NAMESPACE__ . '\Factory\RepositoryStorageInvalidatorFactory',
-    //                __NAMESPACE__ . '\Invalidator\TaxonomyStorageInvalidator'   => __NAMESPACE__ . '\Factory\TaxonomyStorageInvalidatorFactory'
-    //            ]
-    //        ],
-    //        'listens'      => [
-    //            'Versioning\RepositoryManager'         => [
-    //                'checkout' => [
-    //                    __NAMESPACE__ . '\Invalidator\RepositoryStorageInvalidator'
-    //                ]
-    //            ],
-    //            'Taxonomy\Manager\TaxonomyManager'     => [
-    //                'create' => [
-    //                    __NAMESPACE__ . '\Invalidator\TaxonomyStorageInvalidator'
-    //                ],
-    //                'update' => [
-    //                    __NAMESPACE__ . '\Invalidator\TaxonomyStorageInvalidator'
-    //                ]
-    //            ],
-    //            'Navigation\Manager\NavigationManager' => [
-    //                'page.create'      => [
-    //                    __NAMESPACE__ . '\Invalidator\NavigationStorageInvalidator',
-    //                ],
-    //                'page.update'      => [
-    //                    __NAMESPACE__ . '\Invalidator\NavigationStorageInvalidator'
-    //                ],
-    //                'page.remove'      => [
-    //                    __NAMESPACE__ . '\Invalidator\NavigationStorageInvalidator',
-    //                ],
-    //                'parameter.create' => [
-    //                    __NAMESPACE__ . '\Invalidator\NavigationStorageInvalidator',
-    //                ],
-    //                'parameter.update' => [
-    //                    __NAMESPACE__ . '\Invalidator\NavigationStorageInvalidator',
-    //                ],
-    //                'parameter.remove' => [
-    //                    __NAMESPACE__ . '\Invalidator\NavigationStorageInvalidator',
-    //                ],
-    //            ]
-    //        ]
-    //    ]
```

Comment:
Thanks! I noticed that `./CacheInvalidator/src/Invalidator/NavigationStorageInvalidator.php` and other here mentioned files are still in the repo without being called. I guess they can be deleted...